src/pages/installation.md

```diff
@@ -3,58 +3,67 @@ layout: ~/layouts/Main.astro
+- **A terminal** - Astro is mainly accessed by the terminal's Command Line Interface (CLI).
 
-## Recommended Install
+## Astro Installer
+<!-- Feels like this needed stronger prominence in the text than it had previously -->
+```bash
+npm init astro
+```
 
-`npm init astro` is the easiest way to install Astro in a new project. Run this command in your terminal to start our `create-astro` install wizard to walk you through setting up a new project.
+Run this command in your terminal to start our `create-astro` Installer. Letting you walk through setting up a new project.
 
 ```bash
 mkdir <project-name>
 cd <project-name>
 npm init astro
 ```
 
-Follow the CLI instructions to install Astro with one of our official project starter templates. 
+Follow along with the CLI instructions to install Astro with one of our official project starter templates.
 
-Once completed, jump over to our [Quickstart Guide](/quick-start#start-your-project) for a 30-second walkthrough on how to start & build your new Astro project!
+Once completed, jump over to our [QuickStart Guide](/quick-start#start-your-project) for a walk-through on how to **Start & Build** your new Astro project!
 
 ## Manual Install
 
+You can install Astro without the aide of the Astro Installer. Below demonstrates the best way to go about starting your project
+
 ### Set up your project
 
 Create an empty directory with the name of your project, and then navigate into it:
 
 ```bash
+# Note: Replace <project-name> with the name of your project.
 mkdir <project-name>
 cd <project-name>
-# Note: Replace <project-name> with the name of your project.
 ```
 
-Create a new `package.json` file for your project. Astro is designed to work with the npm ecosystem of packages, which is managed in a `package.json` project manifest. If you don't know what the `package.json` file is, we highly recommend you to have a quick read on [the npm documentation](https://docs.npmjs.com/creating-a-package-json-file).
+Astro is designed to work with the entire [npm](https://www.npmjs.com/) ecosystem, which is managed by the `package.json` project manifest.
+
+To create a new [`package.json`](https://docs.npmjs.com/creating-a-package-json-file)file for your project, run the following command:
 
 ```bash
-# This command will create a basic package.json for you
 npm init --yes
 ```
 
-### Install Astro
+Once your project is setup, you should have a directory with a single `package.json` file inside of it. You can now install Astro in your project.
 
-If you've followed the instructions above, you should have a directory with a single `package.json` file inside of it. You can now install Astro in your project. 
+We use [`npm`](https://www.npmjs.com/) in the examples below, but you could also use [`yarn`](https://yarnpkg.com/) or [`pnpm`](https://pnpm.io/) if you prefer an npm alternative. If you're not familiar with either `yarn` or `pnpm`, then we highly recommend sticking with `npm`.
```

Comment:
Why do we highly recommend npm?

Reply:
😅  apparently its easier, that since it comes bundled with node, we can at least be certain that they have it installed on their machine,